api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/DQRForDuplicateFichaResumoDataSet.java

```diff
@@ -0,0 +1,46 @@
+package org.openmrs.module.eptsreports.reporting.library.datasets;
+
+import org.openmrs.PatientIdentifierType;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.eptsreports.reporting.data.converter.GenderConverter;
+import org.openmrs.module.reporting.data.DataDefinition;
+import org.openmrs.module.reporting.data.converter.DataConverter;
+import org.openmrs.module.reporting.data.converter.ObjectFormatter;
+import org.openmrs.module.reporting.data.patient.definition.ConvertedPatientDataDefinition;
+import org.openmrs.module.reporting.data.patient.definition.PatientIdentifierDataDefinition;
+import org.openmrs.module.reporting.data.person.definition.*;
+import org.openmrs.module.reporting.dataset.definition.DataSetDefinition;
+import org.openmrs.module.reporting.dataset.definition.PatientDataSetDefinition;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DQRForDuplicateFichaResumoDataSet extends BaseDataSet {
+
+  public DataSetDefinition constructDataSet() {
+    PatientDataSetDefinition pdd = new PatientDataSetDefinition();
+
+    pdd.setName("EC1");
+
+    PatientIdentifierType identifierType =
+        Context.getPatientService()
+            .getPatientIdentifierTypeByUuid("e2b966d0-1d5f-11e0-b929-000c29ad1d07");
+    DataConverter identifierFormatter = new ObjectFormatter("{identifier}");
+    DataDefinition identifierDef =
+        new ConvertedPatientDataDefinition(
+            "identifier",
+            new PatientIdentifierDataDefinition(identifierType.getName(), identifierType),
+            identifierFormatter);
+    DataConverter formatter = new ObjectFormatter("{familyName}, {givenName}");
+    DataDefinition nameDef =
+        new ConvertedPersonDataDefinition("name", new PreferredNameDataDefinition(), formatter);
+    pdd.setParameters(getParameters());
+
```

Comment:
We can add pid here since I see it on the template:
` patientDataSetDefinition.addColumn("pid", new PersonIdDataDefinition(), "");
`
We should also add this mapping to the template @rupansana12